jdaviz/core/template_mixin.py

```diff
@@ -51,6 +51,12 @@ def show_widget(widget, loc, title):  # pragma: no cover
                            "To learn more, see our documentation at: "
                            "https://jdaviz.readthedocs.io")
 
+    if height is not None:
+        if isinstance(height, int):
+            height = f"{height}px"
+        widget.layout.height = height
+        widget.state.settings['context']['notebook']['max_height'] = height
```

Comment:
I'm not sure this should be in the generic show_widget call or just limited to app.  I think the state that is being set only affects the output cell for the entire app right now (although it might be nice to either obey that or have a separate setting for other widgets).

Reply:
Hm, good point. I can move this into the helper `show` method instead.

Reply:
does the `widget.layout.height` still work, for plugins for example?

Reply:
The plugin tray still fills the whole height of the app, if that's what you're asking.

Reply:
Sorry, I mean would the `widget.layout.height` set the height when you show just a plugin via something like `viz.plugins['Line Analysis'].show(height=50)`, or since that does not listen to context.notebook.max_height, it still expands to show the entire plugin, no matter how tall?

Reply:
Ah, gotcha. I moved this out of here and up into the `helper.show` method, so it's no longer relevant I think.

Reply:
I'd be curious to try it though...